frontend/src/metabase/lib/time-dayjs.ts

```diff
@@ -18,9 +18,7 @@ const DAYLIGHT_SAVINGS_CHANGE_TOLERANCE: Record<string, number> = {
 
 /**
  * This function is used to get a tolerance for the difference between two dates
- * across the daylight savings time change, using dayjs' date.diff method. For
```

Comment:
Not related to the main purpose of this PR, I just noticed I left an incomplete sentence here so I'm removing it.